modules/schema/src/category/table/table-types.ts

```diff
@@ -5,6 +5,7 @@ import type {Batch} from '../common';
 
 /** A general table */
 export interface Table {
+  name?: string;
```

Comment:
Should we keep the name outside the table? 

I am biased towards keeping these basic shapes simple.

Most formats have a single table that does not have a name.

Reply:
I had a slight preference for the name inside the `Table` interface so that there wasn't an extra level of nesting: `Tables.tables.table`

Reply:
Yes but where do we draw the line... there will always be a temptation to slap more and more things inside the returned data types.

Reply:
My slight preference is that an optional `name` is general enough to be in a base `Table` type. In the OGR model, every "layer" (which I think is closest to our `Table`) has a name. For a format like GeoJSON or Shapefile that doesn't support multiple layers, the layer name is the filename. But you still have to supply that name a second time when converting via `ogr2ogr`